mapie/tests/test_time_series_regression.py

```diff
@@ -518,3 +518,25 @@ def test_method_error_in_update(monkeypatch: Any, method: str) -> None:
     with pytest.raises(ValueError, match=r".*Invalid method.*"):
         mapie_ts_reg.fit(X_toy, y_toy)
         mapie_ts_reg.update(X_toy, y_toy)
+
+
+@pytest.mark.parametrize("method", ["enbpi", "aci"])
+@pytest.mark.parametrize("cv", ["split", "prefit"])
+def test_methods_preservation_in_fit(method: str, cv: str) -> None:
+    """Test of enbpi and aci method preservation in the fit MapieRegressor"""
+
+    X_train_val, X_test, y_train_val, y_test = train_test_split(
+        X, y, test_size=0.33, random_state=random_state
+    )
+    X_train, X_val, y_train, y_val = train_test_split(
+        X_train_val, y_train_val, test_size=0.5, random_state=random_state
+    )
+    estimator = LinearRegression().fit(X_train, y_train)
+    mapie_ts_reg = MapieTimeSeriesRegressor(
+        estimator=estimator,
+        cv=cv, method=method
+    )
+    mapie_ts_reg.fit(X_val, y_val)
+    mapie_ts_reg.update(X_test, y_test, gamma=0.1, alpha=0.1)
+    assert mapie_ts_reg.method == method, \
+        f"Expected method {method}, but got {mapie_ts_reg.method}"
```

Comment:
Here, all you have to do is check that the class has the right method. You don't have to handle any exceptions.
```suggestion
    assert mapie_ts_reg.method == method
```